src/main/java/org/cryptomator/frontend/fuse/OpenFile.java

```diff
@@ -108,7 +108,7 @@ public void fsync(boolean metaData) throws IOException {
 	}
 
 	public void truncate(long size) throws IOException {
-		channel.truncate(size);
+		TruncateOrExpandFileChannelUtility.execute(channel, size);
```

Comment:
Just a naming suggestion:

`FileChannelUtil.truncateOrExpand(channel, size)`

Reply:
:+1: applied it in https://github.com/cryptomator/fuse-nio-adapter/pull/66/commits/f41ec90f725cdee3deff9808488619ce864e224a